lib/core/numbers.toit

```diff
@@ -618,19 +620,38 @@ abstract class int extends num:
     else:
       return parse_generic_radix_ radix data from to --on_error=on_error
 
+  static char_to_int_ c/int -> int:
```

Comment:
7 lines up the condition in parse_ should probably be
if not 0 <= from <= to <= data.size

Reply:
or
`if not 0 <= from < to <= data.size`